SeleniumDemoQA/SeleniumDemoQA/Tests/TextBoxTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeleniumDemoQA.Tests
+{
+    public class TextBoxTests: BaseClass
+    {
+        [Test, Description("Enter valid data in input fields, press the submit button and verify the output in the aftersubmit text area")]
+        public void FillAndSubmitTextBoxTests()
+        {
+            _driver.Navigate().GoToUrl("https://demoqa.com/text-box");
+
+            FillInput(By.Id("userName"), "Dog Pulya");
+            FillInput(By.Id("userEmail"), "pulyadog@gmail.com");
+            FillInput(By.CssSelector("#currentAddress.form-control"), "Varash city, Budivelnikiv street, building 3, apartment 333");
```

Comment:
Having  the same data "Varash city, Budivelnikiv street, building 3, apartment 333" for both addresses is bad. You need to be sure that what you input in CurrentAddress is reflected in CurrentAddress and PermanentAddress in PermanentAddress. If data is the same, there can be bugs like:
1) CurrentAddress input goes to both outputs
2) PermanentAddress input goes to both outputs
3) CurrentAddress input goes to PermanentAddress output and vise versa
All these possible bugs will be missed by this test